src/setupTests.js

```diff
@@ -1,3 +1,5 @@
+global.scrollTo = jest.fn();
```

Comment:
Can we just add this to the tests that need it instead of the global test setup file?